api/server/routes/convos.js

```diff
@@ -187,6 +200,34 @@ router.get('/import/jobs/:jobId', async (req, res) => {
     logger.error('Error getting job details', error);
     res.status(500).send('Error getting job details');
   }
+};
+
+// Get the status of an import job for polling
+router.get('/import/jobs/:jobId', jobStatusHandler);
+
+// Get the status of an export job for polling
+router.get('/export/jobs/:jobId', jobStatusHandler);
+
+router.get('/export/jobs/:jobId/conversations.json', async (req, res) => {
+  logger.info('Downloading JSON file');
+  try {
+    //put this in a function
+    const { jobId } = req.params;
```

Comment:
Should we put that in a function?

Reply:
I remember that we wanted to handle user id and job id to make sure we return the appropriate http code